site/content/about.md

```diff
@@ -8,13 +8,13 @@ Publishing a digital garden isn't easy. As [Quartz](https://quartz.jzhao.xyz/not
 
 > There are an overwhelming number of tutorials, resources, and guides for tools like [Notion](https://www.notion.so/), [Roam](https://roamresearch.com/), and [Obsidian](https://obsidian.md/), yet none of them have super easy to use *free* tools to publish that garden to the world.
 
-In our case to easy and free we'd add *elegant* and *customizable*.
+In our case to easy and free we'd add _elegant_ and _customizable_.
```

Comment:
Hmmm. I much prefer `*elegant* to _elegant_`. Can we tweak eslint on our markdown. (for now not a big deal)